openvino/cc/scripts/ccheader.py

```diff
@@ -0,0 +1,155 @@
+#!/usr/bin/env python3
+
+import argparse, csv
+from pathlib import Path
+
+Domain = ['CC0_',
```

Comment:
Can we have something more intelligible here? I mean use words `SIMPLE`, `SWITCH`, `FACTORY` instead of enumeration?

Reply:
It's internal implementation details. Anyway I'll rename it.
Renamed.